src/.env

```diff
@@ -0,0 +1,3 @@
+REACT_APP_HOST=wss://cc-edapps.calibrateconsulting.com
+REACT_APP_APP1 =a8def80d-0228-4837-ad84-47f8bf8bfb0a
+REACT_APP_APP2= 25bbde2e-79eb-4eab-9c12-4ab56594a738
```

Comment:
```suggestion
REACT_APP_APP2=25bbde2e-79eb-4eab-9c12-4ab56594a738
```

Reply:
This app id is different from the one in the existing code? Is that intentional?